nav2_smac_planner/test/test_smac_2d.cpp

```diff
@@ -77,6 +77,15 @@ TEST(SmacTest, test_smac_2d) {
   } catch (...) {
   }
 
+  // corner case where the start and goal are on the same cell
+  goal.pose.position.x = 0.01;
+  goal.pose.position.y = 0.01;
+
+  try {
+    planner_2d->createPlan(start, goal, dummy_cancel_checker);
```

Comment:
This shouldn't throw, right? We should get a single-point path that we need to check in the return for with `EXPECT_TRUE` on the `size()`